NEWS.md

```diff
@@ -1,8 +1,15 @@
 # admiralmetabolic 0.1.0
 
-- Initial package release focused on `<<ADD INITIAL SCOPE>>`
+- Initial package release mainly focused on obesity therapeutic area
+(e.g. functions for Waist to Hip Ratio, Waist to Height Ratio).
 
-Note that after first release you should then start maintaining this `NEWS.md` file for every user-facing update.
-You should categorize the updates under title sections for New Features, Updates of Existing Functions,
-Breaking Changes, Documentation, Various.
-Each update should also reference the original issue by adding at the end: (#ISSUE-NUMBER)
+## Functions
+
+- The new function "'derive_param_waisthip'" is provided to derive parameters for Waist to Hip Ratio. (#33 &  #52)
+- The new function "'derive_param_waisthgt'" is provided to derive parameters for Waist to Height Ratio. (#33 & #52)
+
+## Documentation
+
+- New vignette "Creating a Metabolic ADVS ADaM" has been created. (#28) 
+- New vignette "Creating a Control of Eating Questionnaire ADaM" has been created. (#53)
+- New template scripts for ADVS "'ad_advs.R'" has been created. (#35)
```

Comment:
```suggestion
- Vignette "Creating a Metabolic ADVS ADaM". (#28) 
- Vignette "Creating a Control of Eating Questionnaire ADaM". (#53)
- ADVS template scripts `ad_advs.R` which creates a Vital Sign Analysis Dataset with metabolic specific derivations. (#35)
```

Reply:
Hi @AndersAskeland , I updated quotations and wordings.
Could you take a look again?